atom/browser/ui/webui/pdf_viewer_ui.cc

```diff
@@ -187,9 +187,9 @@ class PdfViewerUI::ResourceRequester
 
     base::PostTaskWithTraits(
         FROM_HERE, {BrowserThread::UI},
-        base::Bind(&CallMigrationCallback<StreamResponseCallback>,
-                   base::Passed(&stream_response_cb_),
-                   base::Passed(&stream_info_)));
+        base::BindRepeating(&CallMigrationCallback<StreamResponseCallback>,
```

Comment:
this _might_ be able to be a OnceCallback? i wasn't quite sure here.

Reply:
yeah, for `base::PostTask`, usually only `OnceCallback` needs to be passed.